src/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classnames from "classnames";
+
+export type BreadcrumbItemProps = React.DetailedHTMLProps<React.LiHTMLAttributes<HTMLLIElement>, HTMLLIElement> & {
```

Comment:
Remove the detailed html props and include only the props which are being used in this component

Reply:
I am spreading the `{...props}` .. all detailed HTML props can be used

Reply:
Ok yes I missed that